oci/client/list.go

```diff
@@ -37,6 +37,9 @@ type ListOptions struct {
 	SemverFilter string
 	// RegexFilter contains a regex that tags will be filtered by.
 	RegexFilter string
+	// IncludeCosignSignatures can be used to include cosign signature tags
+	// (i.e. ".sig" suffix) in the list, as these are excluded by default.
+	IncludeCosignSignatures bool
```

Comment:
I would rename this to `IncludeCosignArtifacts` and in the exclusion list please add `.att`.

As seen here, there are 2 artifacts for each container image, `sig` and `att`: https://github.com/fluxcd/kustomize-controller/pkgs/container/kustomize-controller

Reply:
Sure! They also use '.sbom', should I add it also?

Reply:
Yes please